cmd/up/up.go

```diff
@@ -352,6 +352,12 @@ func (up *upContext) activate(autoDeploy, build, resetSyncthing bool) {
 
 		if prevError != nil {
 			if up.shouldRetry(prevError) {
+				if !up.Dev.PersistentVolumeEnabled() {
+					if err := pods.Destroy(up.Pod, up.Dev.Namespace, up.Client); err != nil {
```

Comment:
how much time does this add to the retry cycle? If it's more than a couple of seconds (e.g. due to the image pulling) maybe it's better to not support this retry mode for ephermeral dev envs?  (or maybe we can start optimizing so we don't pullAlways the okteto images as part of this change?